github/__init__.py

```diff
@@ -227,12 +227,9 @@ async def repositories(self) -> list[Repository]:
                 except KeyError:
                     return result
 
-    async def get_check_suites(self, ref: str) -> list:
-        # https://docs.github.com/en/rest/reference/checks#list-check-suites-for-a-git-reference
-        response = await self.get(f"/repos/{self.repo_path}/commits/{ref}/check-suites")
+    async def get_check_runs(self, ref: str, per_page: int = 100, page: int = 1) -> tuple[list, int]:
```

Comment:
I would probably make this 
```suggestion
    async def get_check_runs(self, ref: str) -> tuple[list, int]:
```
and move the paging code into this function. Then it is easy and clear to use imo.

Reply:
I don't get it. You can't paginate without providing `per_page` and `page` with GitHub request.

Reply:
Yeah, it would do the paging internally in the function and return all check runs. Then you don't need the parameters.

Reply:
Ah, I got it. Good idea!

Reply:
Done.